<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
border:none;
	font: 9pt &quot;微软雅黑&quot;;
}
QWidget{
border-top-left-radius:15px;;
border-top-right-radius:15px;
border-bottom-left-radius:15px;
border-bottom-right-radius:15px;
}
QPushButton:hover
{
background-color :rgb(255, 170, 127);
 border-radius: 4px; 
}
QWidget#Widget{border-image: url(:/src/background/风绽-美叶 (10).jpg);}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="MusicFile">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>换源</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="MusicFile_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>列表</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="MusicLyric">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>歌词</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="playset">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>模式</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="MusicBack">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>上一首</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="MusicStop">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>暂停</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="MusicSpeed">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>下一首</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="MusicVolume">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>音量</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="MusicList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>461</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget{

background-color: argb(200, 100, 255, 200);
	color: rgb(0, 85, 127);
font: 10pt &quot;微软雅黑&quot;;
}</string>
   </property>
  </widget>
  <widget class="SliberBar" name="Progressbar">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal{
    background:#4b4b4b;
    height:6px;
    border-radius:5px;
}
QSlider::add-page:horizontal{
    background:#c5c5c5;
    height:6px;
    border-radius:5px;
}
QSlider::sub-page:horizontal{
    background:rgba(2, 183, 196, 0.75);
    height:6px;
    border-radius:5px;
}
QSlider::handle:horizontal{
    width:25px;
    margin-top: -17px;
    margin-bottom: -17px;
    border-radius: 5px;
    background: rgba(255, 255, 255, 0.85);
}
QSlider::handle:horizontal:pressed, QSlider::handle:horizontal:hover{
    width:25px;
    border-radius:5px;
    background: #FFFFFF;
}
QSlider::add-page:horizontal:disabled{
    background:#616161;
}
QSlider::sub-page:horizontal:disabled{
    background:rgba(141, 141, 141, 0.75);
}
QSlider::handle:horizontal:disabled{
    background: rgba(112, 112, 112, 0.85);
}</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="SliberBar" name="Volumebar">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>22</width>
     <height>71</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-4</number>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="invertedControls">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="time">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>分</string>
   </property>
  </widget>
  <widget class="QLabel" name="timeAll">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>210</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="Volumevalue">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>120</y>
     <width>47</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>71</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="Title">
      <property name="text">
       <string>歌曲:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Author">
      <property name="text">
       <string>艺术家:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Disc">
      <property name="text">
       <string>唱片集:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Format">
      <property name="text">
       <string>格式:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="LyircClass" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>461</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <item>
     <widget class="QLabel" name="Lyric_1">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Lyric_2">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Lyric_3">
      <property name="styleSheet">
       <string notr="true">QLabel{
//font: 25 10pt &quot;Microsoft YaHei&quot;;                        //字体形状大小
border-radius: 8px;                                            //圆角设置
 
//标签颜色

 //字体颜色
text-align: center;

}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Lyric_4">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Lyric_5">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Lyric_6">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>181</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="TirleLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="AuthorLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="DiscLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="FormatLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="Picture">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyQLabel" name="Spectrum">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>250</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>widget</zorder>
  <zorder>widget_2</zorder>
  <zorder>Volumebar</zorder>
  <zorder>Progressbar</zorder>
  <zorder>time</zorder>
  <zorder>MusicList</zorder>
  <zorder>timeAll</zorder>
  <zorder>Volumevalue</zorder>
  <zorder>Picture</zorder>
  <zorder>Spectrum</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SliberBar</class>
   <extends>QSlider</extends>
   <header location="global">sliberbar.h</header>
  </customwidget>
  <customwidget>
   <class>LyircClass</class>
   <extends>QWidget</extends>
   <header location="global">lyircclass.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyQLabel</class>
   <extends>QLabel</extends>
   <header location="global">myqlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
